### DISCUSSION TITLE
Idea for a new layout?

### DISCUSSION THREAD

**Discussion by nbouvrette** [orestbida/cookieconsent#240] [Ideas]
We've started using Cookie Conscent on our site and it looks like the best option around (great work).

One thing that we noticed is that the "bar" layout was using a lot of real estate on our page. We were wondering if there would be interest if we would do a PR on some sort of "thin-bar" layout? Could look like this:

<img width="996" alt="image" src="https://user-images.githubusercontent.com/6453322/154828243-c4ef8663-e010-4ac5-8c7a-e52af098737f.png">

**Comment by orestbida** [comment]
@nbouvrette,

Love it! A PR is very welcome!
